NEXT-12776 -  [MIGRATE] Administration: Using mixins (#175)
shopware/docs: resources/references/core-reference/administration-reference/mixins.md

# Mixins reference

This is an overview of all the mixins provided by the Shopware 6 Administration. 

> Mixins in the Shopware 6 Administration, are essentially the same as normally in Vue. 

Reviewer: ```suggestion
Mixins in the Shopware 6 Administration are essentially the same in default Vue.
```

They behave generally the same as they do in Vue normally, differing only in the registration and the way mixins are included in a component.
Learn more about them in the official [Vue documentation](https://vuejs.org/v2/guide/mixins.html).

> If u want to learn how to use them in your plugin, take a look at this guide [here](./../../../../guides/plugins/plugins/administration/using-mixins.md)

Reviewer 2: I like it informal, but I assume we aren't allowed to use that in our docs. ;)

```suggestion
If you want to learn how to use them in your plugin, take a look at this guide [here](./../../../../guides/plugins/plugins/administration/using-mixins.md)
```

> or if you want to register your own take a look at this [guide](./../../../../guides/plugins/plugins/administration/add-mixins.md)

Reviewer: ```suggestion
or if you want to register your own mixin take a look at this [guide](./../../../../guides/plugins/plugins/administration/add-mixins.md)
```

> ## Overview of all the mixins

Reviewer: Please order the table descending by mixin name A-Z
